include/elm/inhstruct/DLList.h

```diff
@@ -31,31 +31,38 @@ class DLNode {
 	friend class DLList;
 	DLNode *nxt, *prv;
 public:
+	DLNode() : nxt(nullptr), prv(nullptr) {}
+
 	inline DLNode *next(void) const { return nxt; }
 	inline DLNode *previous(void) const { return prv; }
-	inline bool atBegin(void) const { return prv == 0; }
-	inline bool atEnd(void) const { return nxt == 0; }
+	inline bool atBegin(void) const { return prv == nullptr; }
+	inline bool atEnd(void) const { return nxt == nullptr; }
```

Comment:
This change does nothing unless you are on an esoteric setup where nullptr =/= 0.